### PR TITLE
Add PCA9555

### PR DESCRIPTION
**Pull request by luis109**
datasheet: https://www.nxp.com/docs/en/data-sheet/PCA9555.pdf

Screenshot:
pca

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
- [x] An example screenshot image is very helpful
- [x] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [x] If there are matching footprint PRs, provide link(s) as appropriate
- [x] Check the output of the Travis automated check scripts - fix any errors as required
- [x] Give a reason behind any intentional library convention rule violation.

### CODE REVIEW

**Comment by jneiva08** [comment]
it appears to not have anything wrong

**Comment by luis109** [comment]
Made requested changes. 
Please verify if it is in accordance with your comment. 

Thank you.

File: Interface_Expansion.lib [hunk 1]
```diff
@@ -896,6 +896,123 @@ X SC2 9 400 300 100 L 50 50 1 1 O
...
 #
+# PCA9555D
+#
+DEF PCA9555D U 0 20 Y Y 1 F N
+F0 "U" -350 950 50 H V C CNN
+F1 "PCA9555D" 250 950 50 H V C CNN
+F2 "" 0 0 50 H I C CNN
+F3 "" 0 0 50 H I C CNN
+$FPLIST
+ SOIC*W*7.5x15.4mm*P1.27mm
```
Comment (chschlue) [inline Interface_Expansion.lib:907]: Should be `SOIC*7.5x15.4mm*P1.27mm*`

File: Interface_Expansion.lib [hunk 2]
```diff
@@ -896,6 +896,123 @@ X SC2 9 400 300 100 L 50 50 1 1 O
...
+#
+# PCA9555PW
+#
+DEF PCA9555PW U 0 20 Y Y 1 F N
+F0 "U" -350 950 50 H V C CNN
+F1 "PCA9555PW" 250 950 50 H V C CNN
+F2 "" 0 0 50 H I C CNN
+F3 "" 0 0 50 H I C CNN
+$FPLIST
+ TSSOP*4.4x7.8mm*P0.65mm
```
Comment (chschlue) [inline Interface_Expansion.lib:985]: Should be `TSSOP*4.4x7.8mm*P0.65mm*`

**Comment by luis109** [comment]
Made requested changes.

Thank you @chschlue

**Pull request merged**